CRM/Core/BAO/UFGroup.php

```diff
@@ -1399,6 +1399,13 @@ public static function usedByModule($id) {
    *
    */
   public static function del($id) {
+
+    $group = new CRM_Core_DAO_UFGroup();
+    $group->id = $id;
+    $group->fetch();
+    $params = $group->toArray();
```

Comment:
We don't normally load from the db just to pass out to hooks -the db load can be done in the hook if it wants it.

I took a look at the other delete hooks & realised they are a bit of a dogs breakfast so I put up this PR to make the laster parameter optional & clean them up a bit

https://github.com/civicrm/civicrm-core/pull/19059

Reply:
@lucky091588 Eileen's commit #19059 means we'll need to rebase this PR and probably resolve some git merge conflicts. Let's talk about that offline.